### PR TITLE
Add RV, redshift columns to target summary table.

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: explore/src/main/scala/explore/targeteditor/TargetSummaryTable.scala [hunk 1]
```diff
@@ -72,6 +73,9 @@ object TargetSummaryTable {
     "epoch"        -> "Epoch",
     "pmra"         -> "µ RA",
     "pmdec"        -> "µ Dec",
+    "rv"           -> "RV",
```
Comment (cquiroz) [inline explore/src/main/scala/explore/targeteditor/TargetSummaryTable.scala:76]: Do we want to display the three versions in the table?

**Review by toddburnside — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/targeteditor/TargetSummaryTable.scala (hunk 1 above)
Comment (rpiaggio) [reply explore/src/main/scala/explore/targeteditor/TargetSummaryTable.scala:76]: As per Andy's request, yes. We only show `RV` initially.

**Review by cquiroz — APPROVED** [review]
LGTM, I'd like at a later date review alternatives to support conversions in a single column but this will do for now

**Pull request merged**